src/components/CreateMemorandum/UploadMemorandum.vue

```diff
@@ -61,7 +61,7 @@ export default class UploadMemorandum extends Vue {
   padding: 1.25rem;
 
   .upload-memorandum-title {
-    padding-left: 0.5rem;
+    color: $gray9;
```

Comment:
I updated the colour here, and font weight in the template, to be consistent between components, and as requested by Tracey.